dev/Repeater/TestUI/Samples/ScaleAnimatedVerticalListDemo.xaml

```diff
@@ -0,0 +1,21 @@
+﻿<Page
+    x:Class="MUXControlsTestApp.Samples.ScaleAnimatedVerticalListDemo"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:mux="using:Microsoft.UI.Xaml.Controls"
+    Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">
+
+    <Grid>
+        <ScrollViewer x:Name="sv" Width="200" Height="150"  VerticalScrollBarVisibility="Hidden">
+            <mux:ItemsRepeater x:Name="repeater">
+                <mux:ItemsRepeater.ItemTemplate>
+                    <DataTemplate>
+                        <StackPanel>
```

Comment:
In practice items tend to contain more content than a single item so its likely they'd be wrapped in a Panel, but for the sample it would be great if it wasn't required.  Alternatively, someone might create a UserControl that is the sole item declared in the DataTemplate.  Would that still need to be wrapped in a Panel to animate the offset or is it possible to just target the Content?  @gegao18, any alternative recommendations?  @ranjeshj, if we need something to act as the wrapper then would Canvas be better (lighter-weight) than StackPanel?

Reply:
If there is another panel, we could use that. In the trivial case here we need one just to animate offset. I don't know if there is a big difference between stack and canvas - is there any data to support that ?

Reply:
removed the extra panel and used CenterPoint instead.